### DISCUSSION TITLE
Categorzing liability transactions

### DISCUSSION THREAD

**Discussion by venkivijay** [firefly-iii/firefly-iii#6452] [Firefly III]
From the docs, I understand that for liabilities I have to create two transactions. How do I categorize these two transactions? If I add them in the same category then that category is showing double the amount. If I don't categorize one of the transactions, it goes to an uncategorised section which I don't like (as it's messing up the charts).

**Comment by JC5** [comment]
This is something I intend to fix in https://github.com/firefly-iii/firefly-iii/issues/6456
